src/setupTests.js

```diff
@@ -36,3 +36,6 @@ Object.defineProperty(window, 'scrollTo', {
 // own label.
 base.randomId = (prefix) =>
   `${prefix}-${(Math.random() + Math.random() + 1).toString(36).substring(2)}`;
+
+// Storage tests take over 5 seconds
+jest.setTimeout(10000);
```

Comment:
Which tests take >5 seconds? If there are only a few, just change the timeout for those tests or test suites.

Reply:
Pretty much all storage tests that interact with the back-end.
I could potentially go and set it one by one if you think it's better?

Reply:
That's unfortunate -- let's set the timeout on the storage tests suites (for whole `describe` blocks).

Reply:
Ok, once I connected the actual emulator, the tests became fast, so I dropped this line.